dell/inspiron/7460/default.nix

```diff
@@ -0,0 +1,31 @@
+{lib, ...}:
+
+with lib;
+
+{
+  imports = [
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+    ../../../common/gpu/intel/kaby-lake
+    ../../../common/gpu/nvidia/maxwell
+    ../../../common/gpu/nvidia/prime.nix
+  ];
+
+  # Essential Firmware
+  hardware.enableRedistributableFirmware = mkDefault true;
```

Comment:
This is also part of nixos-generate-config through `imports = [ (modulesPath + "/installer/scan/not-detected.nix") ];`

Reply:
```suggestion
```